device/mellanox/x86_64-mlnx_msn2740-r0/sensors.conf

```diff
@@ -74,3 +74,39 @@ bus "i2c-2" "i2c-1-mux (chan_id 1)"
         label fan2 "Chassis Fan Drawer-2"
         label fan3 "Chassis Fan Drawer-3"
         label fan4 "Chassis Fan Drawer-4"
+        ignore temp2
```

Comment:
There are 32 ports in 2740, but there are 35 ignore here

Reply:
changed